### PR TITLE
Quick and dirty patch to show user cards in the connected peers.

### PR DESCRIPTION
**Pull request by jjeffryes**
- shows user cards in the connected peers view so it's clear which ones are working profiles
- this is for testing in milestone 1, not production use.

### CODE REVIEW

File: js/templates/connectedPeersPage.html [hunk 1]
```diff
@@ -1,11 +1,23 @@
 <div class="pad tx4">
+  <% console.log(ob) %>
```
Comment (rmisio) [inline js/templates/connectedPeersPage.html]: Stray console statement here.

File: js/templates/connectedPeersPage.html [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+    <div class="js-morePeers hide">
+      <hr class="clrBr">
+      <a class="btn floR clrBr clrP js-morePeersBtn">Load More</a>
+    </div>
+  </div>
+
+  <% if (!ob.peers.length) { %>
     <p><%= ob.polyT('connectedPeersPage.noResults') %></p>
   <% } %>
+
```
Comment (rmisio) [inline js/templates/connectedPeersPage.html]: Can we clean up this white space here please?

File: js/templates/connectedPeersPage.html [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
   <p><em><%= ob.polyT('connectedPeersPage.infoMessage') %></em></p>
-  <% if (ob.peers.length) { %>
-    <ul>
-    <% ob.peers.forEach(peer => print(`<li><a href="#${peer}">${peer}</a></li>`)) %>
-    </ul>
-  <% } else { %>
+
+  <div class="pageContent">
+    <div class="userPageFollow flexRow js-peerWrapper">
+
```
Comment (rmisio) [inline js/templates/connectedPeersPage.html]: Can we clean up this empty line?

File: js/views/ConnectedPeersPage.js [hunk 4]
```diff
@@ -11,6 +12,48 @@ export default class extends baseVw {
...
+
+  get $morePeers() {
+    return this._$morePeers ||
+        (this._$morePeers = this.$('.js-morePeers'))
+  }
+
+  loadPeers() {
+    if (this.peers.length > this.peersLoaded) {
+      this.peers.slice(this.peersLoaded, this.peersLoaded + this.peersIterator).forEach((peer) => {
+        console.log(peer)
```
Comment (rmisio) [inline js/views/ConnectedPeersPage.js]: Stray console statement here.

File: js/views/ConnectedPeersPage.js [hunk 5]
```diff
@@ -11,6 +12,48 @@ export default class extends baseVw {
...
+        const user = this.createChild(userShort, {
+          guid: peer,
+        });
+        this.$peerWrapper.append(user.render().$el);
+      });
+      this.peersLoaded += this.peersIterator;
```
Comment (rmisio) [inline js/views/ConnectedPeersPage.js]: There's a good chance the last page will not be a full page, so for peersLoaded to be accurate, you would want to add the amount you sliced above.

But... it is inconsequential here. For what we are using peersLoaded for, it doesn't matter if it's wrong, as long as it's over not under.

File: js/views/ConnectedPeersPage.js [hunk 6]
```diff
@@ -11,6 +12,48 @@ export default class extends baseVw {
...
+  }
+
+  loadPeers() {
+    if (this.peers.length > this.peersLoaded) {
+      this.peers.slice(this.peersLoaded, this.peersLoaded + this.peersIterator).forEach((peer) => {
+        console.log(peer)
+        const user = this.createChild(userShort, {
+          guid: peer,
+        });
+        this.$peerWrapper.append(user.render().$el);
```
Comment (rmisio) [inline js/views/ConnectedPeersPage.js]: Could we add these to a document fragment and then add the whole fragment instead of one by one to the DOM?

**Comment by rmisio** [comment]
Could you also please fix the linting errors?

File: js/views/ConnectedPeersPage.js (hunk 5 above)
Comment (jjeffryes) [inline js/views/ConnectedPeersPage.js]: I'll change the variable name, it's not actually intended to track the amount loaded, just the amount it's attempting to load. How it works here is intentional.

File: js/views/ConnectedPeersPage.js (hunk 6 above)
Comment (jjeffryes) [inline js/views/ConnectedPeersPage.js]: Yup.

**Comment by jjeffryes** [comment]
I'll clean it up. It was a spur of the moment thing on my older computer (I got inspired while checking an old Bitcoin wallet), that computer didn't have linting set up the way we do now.

**Comment by tenthhuman** [comment]
That is super nice. 
People can explore the network like this.
It would be cool if this was more accessible. Instead of hidden in settings.

**Comment by tenthhuman** [comment]
This closes #61.

**Pull request merged**